src/models/data.h

```diff
@@ -46,23 +46,40 @@ QString prettifySymbol(const QString& symbol);
 
 struct Symbol
 {
+    static bool prettifySymbol;
```

Comment:
wrong place to store this, introduce a QObject Settings singleton with a getter and setter for this property as well as a notify signal which can then be connected to in the models. The models then need to trigger a dataChanged signal on the respective Name column